pyproject.toml

```
@@ -60,3 +61,24 @@ requires = [
     "versioneer-518"
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+norecursedirs= [
+    ".*", "*.egg*", "build", "dist", "conda-recipe",
+]
+xfail_strict = true
+addopts = [
+    "--disable-warnings",
+    "--ignore setup.py",
```

Don't you need to make them separate strings?
```suggestion
    "--ignore", "setup.py",
```

Same for others.

---

or may be just for the last line, I'm not sure, but the workflow is failing